resources/views/user/roles.blade.php

```diff
@@ -0,0 +1,16 @@
+{{-- This should be in a tr --}}
+<td>
+    @foreach($user->roles as $role)
+    <span class="new badge {{ $role->color() }}" data-badge-caption="">{{ $role->name() }}
+        @if($role->pivot->object_id)
+            : {{ $role->object()->name }}
+        @endif
+    </span>
+    @endforeach
+</td>
+<td>
+    <a href="{{ route('admin.permissions.show', $user->id) }}"
+        class="btn-floating waves-effect waves-light right" @cannot('viewPermissionFor', $user) disabled @endcan>
+        <i class="material-icons">edit</i>
+    </a>
+</td>
```

Comment:
I think we should add this to the user's page so everyone could see their roles.